README.md

```diff
@@ -1,2 +1,16 @@
 # credentials-io
 Library for reading/writing credentials
+
+## API
+
+### verify
```

Comment:
We'll want to also specify the arguments to each of these functions and the expected return type or what is sent to the callback.